guides/user/managed-delivery/intergate-your-CI/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: single
+title:  "Intergate your CI"
+sidebar:
+  nav: guides
+redirect_from: /reference/managed-delivery/intergate-your-CI
```

Comment:
you don't need this because the page was never somewhere else. this is just for if it moved, which some of the other pages did

Reply:
removed it, thanks